src/imeta.cpp

```diff
@@ -1269,23 +1267,6 @@ parse_program_options(int _argc,
     }
 }
 
-int do_interactive() {
-    while ( true ) {
-        char *cmdToken[MAX_CMD_TOKENS];
-        int num_tokens = getInput( cmdToken, MAX_CMD_TOKENS );
```

Comment:
Looks like `getInput` is unused now. If that's true, let's remove that.

If there's anything else that's not used, remove that too.